examples/multicached_decorator.py

```diff
@@ -11,15 +12,15 @@
     'd': "W"
 }
 
-cache = Cache(Cache.REDIS, namespace="main", client=redis.Redis())
+cache = RedisCache(namespace="main", client=redis.Redis())
 
 
-@multi_cached("ids", cache=Cache.REDIS, namespace="main", client=cache.client)
+@multi_cached(cache=cache, keys_from_attr="ids")
```

Comment:
I'm just going to drop the keyword from the examples.
```suggestion
@multi_cached(cache, keys_from_attr="ids")
```